src/components/profile/Profile.jsx

```diff
@@ -0,0 +1,161 @@
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import './Profile.scss';
+import Nav from '../presentation/NavBar';
+import TrendingTags from '../presentation/TrendingTags';
+import ProfileCard from './ProfileCard';
+import EditProfileModal from './EditProfileModal';
+import BigArticleCard from '../presentation/BigArticleCard';
+import SmallArticleCard from '../presentation/SmallArticleCard';
+import UserTopArticles from './UserTopArticles';
+import Footer from '../presentation/Footer';
+import { getProfile } from '../../actions/profile';
+
+/**
+ * @class
+ * @description - Profile component
+ * @returns {JSX} - rendered output
+ */
+export class Profile extends Component {
+  /**
+   * @constructor
+   * @param {*} props - react props
+   * @returns {void}
+   */
+  constructor(props) {
+    super(props);
+    this.onEditProfile = this.onEditProfile.bind(this);
```

Comment:
Hey, @oluwajuwon, I fell back to manually binding here because, for some reasons, I couldn't `spyOn` the method while testing when it was an elegant class property.

Reply:
oh! alright then 👍